src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/ComputeBuilder.java

```diff
@@ -166,13 +166,12 @@ void associateProviderMixin(MixinService mixinService,
         for (String mixinName : attributes.keySet()) {
```

Comment:
I think that it cannot be refactored with lambda due to the exception thrown. If you disagree, it will be a pleasure to discuss about a solution for that issue.